### PR TITLE
Add 2019-08-01 meeting notes

### PR DESCRIPTION
**Pull request by BrunoScheufler**
Today was a great and productive meeting, I've had a great time taking notes! I've formatted and added the notes of today's working group meeting!

@leebyron great explanation of the lexing internals! I'm not sure if this the right time but I'd love to extend my support to the working group beyond taking notes to possibly helping out with chores on this repo, keeping track of all meeting-related documents, and more things that might help out.

For me it would be easier if I could add reviewers and possibly use the actual repository as the foundation instead of my fork, do you think we could make something happen in this regard?

### CODE REVIEW

File: notes/2019-08-01.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- Lee: Great questions. I don’t have any satisfying answers at the moment. If you are eager on takign this on, it seems like that’s the missing piece. I think there’s some ambiguity why we can’t make it run on all platforms at once, why would you build this on top of your engine when there’s only some tests but then why would you contribute to graphql cats? I think in the beginning we just have to force this through so whoever has the time and passion to pull this off might steer it. People that are new to the ecosystem might question if the tool will work for them would appreciate graphql cats.
+- Michael: The only blocker is that I’m not sure who owns graphql cats right now. If we were to put it into a place where people had to write tests upon changes to the spec. There are definitely rules we’d have to fix but overall it’s a great idea.
+- Lee: Does anybody know who else might have access to GraphQL cats?
```
Comment (joshprice) [inline notes/2019-08-01.md]: I own the GraphQL CATS org and repo. Would love to reboot this project and have it be a first class citizen along with the spec.

Comment (joshprice) [reply notes/2019-08-01.md]: Please let me know how I can help!

Comment (BrunoScheufler) [reply notes/2019-08-01.md]: @leebyron

File: notes/2019-08-01.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# GraphQL Working Group Meeting 2019-08-01
+
+[Agenda](https://github.com/graphql/graphql-wg/blob/master/agendas/2019-08-01.md)
+
+## Action Items
+
+- [ ] **Unclaimed** Add maintainer list for relevant and breaking changes, send to reference library maintainers
```
Comment (IvanGoncharov) [inline notes/2019-08-01.md]: I can take this item and compile this list for the next WG.

**Comment by IvanGoncharov** [comment]
I merged the agenda.

@BrunoScheufler @dblandin It would be great to have new action items submitted as issues:
https://github.com/graphql/graphql-wg/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Action+item+%3Aclapper%3A%22
Do you want to work on this?
and also create an issue template for action items:
https://github.com/graphql/graphql-wg/issues/211#issuecomment-513027534

> For me it would be easier if I could add reviewers and possibly use the actual repository as the foundation instead of my fork, do you think we could make something happen in this regard?

I think it would be great to have couple maintainers for this repo and since you and @dblandin already doing most of work in this repo it makes sense to just give you commit rights. Since I already merged this PR I don't want this question to stay answered so sent an email to Lee.

File: notes/2019-08-01.md (hunk 1 above)
Comment (leebyron) [reply notes/2019-08-01.md]: Awesome! Give me a moment to figure things out there. @joshprice let me know an email to contact you at and I'll fill you in on the details.

**Comment by leebyron** [comment]
I'll look into expanding repo maintenance!

**Pull request merged**